base/libgit2/blob.jl

```diff
@@ -9,6 +9,12 @@ function rawcontent(blob::GitBlob)
     copy(unsafe_wrap(Array, ptr, (length(blob),), false))
 end
 
+"""
+Fetches the contents of the `GitBlob` `blob`. If the `blob` contains
```

Comment:
Since we have formal docs for LibGit2 now, this should probably be formatted as a typical docstring with a signature and whatnot.